OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -223,6 +230,9 @@ public void Target()
 			if (power == null)
 				return;
 
+			if (!HasSufficientFunds(power))
+				return;
+
```

Comment:
You should be able to select this and hover over your target area, waiting for the cash to be available, so i would not check this in here.